src/Symfony/Component/Routing/Attribute/Route.php

```diff
@@ -46,6 +46,7 @@ public function __construct(
         private array $requirements = [],
         private array $options = [],
         private array $defaults = [],
+        private array $alias = [],
```

Comment:
```suggestion
        private array $aliases = [],
```
For consistency with $defaults, $options, etc.